### PR TITLE
4.17.1.0: release notes updates

### PR DESCRIPTION
**Pull request by shwstppr**

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/releasenotes/about.rst [hunk 1]
```diff
@@ -17,6 +17,22 @@
 What's New in |release|
 =======================
 
+Apache CloudStack |release| is a 4.17 LTS minor release with over 150 fixes and
+improvements since the 4.17.0.0 release. Highlights include:
+
+• Imporvement for System VM storage migration on KVM
```
Comment (rohityadavcloud) [inline source/releasenotes/about.rst:23]: Typo

File: source/releasenotes/changes.rst [hunk 2]
```diff
@@ -13,6 +13,336 @@
...
+===================================
+
+Apache CloudStack uses GitHub https://github.com/apache/cloudstack/milestone/25?closed=1
+to track its issues.
+
+.. cssclass:: table-striped table-bordered table-hover
+
+
++-------------------------+----------+---------------+----------+------------------------------------------------------------+
+| Version                 | Github   | Type          | Priority | Description                                                |
```
Comment (rohityadavcloud) [inline source/releasenotes/changes.rst:26]: We can remove the type and priority columns if they're empty (maybe fix the py script that generates the table)?

**Review by rohityadavcloud — COMMENTED** [review]
LGTM, need to recheck the what's new in 4.17.1.0 list and the upgrade notes but otherwise LGTM. Systemvmtemplate seeding happens automatically - so we can assess if we recomend not seeding hte template prior to upgrade (i..e the wordings make sense?)

File: source/upgrading/index.rst [hunk 3]
```diff
@@ -44,6 +44,7 @@ Contents:
  
    upgrade/mysql
    upgrade/valid_source
+   upgrade/upgrade-4.17
```
[the remark this answers is not in the document]
Comment (shwstppr) [reply source/upgrading/index.rst:47]: I guess not. Similar change for 4.16.1 
https://github.com/apache/cloudstack-documentation/commit/844f992db15dc1053f3f2828ae6f5f7512457303#diff-e3196f365d0a93cdffd3ff9f8fe0be8eb7cbc4ff756de966bb24cd5b2e299321R47

File: source/releasenotes/about.rst (hunk 1 above)
Comment (shwstppr) [reply source/releasenotes/about.rst]: Fixed

File: source/releasenotes/changes.rst (hunk 2 above)
Comment (shwstppr) [reply source/releasenotes/changes.rst:26]: Tried but it messes with formatting. Need to spend more time. For later

**Comment by shwstppr** [comment]
@blueorangutan docbuild

File: source/upgrading/index.rst (hunk 3 above)
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply source/upgrading/index.rst:47]: I think it signifies upgrade *from* so were fine

**Review by DaanHoogland — APPROVED** [review]

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Review by rohityadavcloud — APPROVED** [review]

**Pull request merged**